src/main/java/pl/fracz/mcr/source/NoSelectedLineException.java

```diff
@@ -0,0 +1,6 @@
+package pl.fracz.mcr.source;
+
+import pl.fracz.mcr.comment.CommentNotAddedException;
+
+public class NoSelectedLineException extends CommentNotAddedException {
```

Comment:
Hierarchia dziedziczenia wyjątków co najmniej dziwna w tym wypadku, ponieważ opisują dwie sytuacje.
Może to spowodować problemy jak ktoś tego nie świadomy napisze catch'e w złej kolejności.